src/components/Header.tsx

```diff
@@ -21,12 +21,24 @@ export default function Header() {
 
     useEffect(() => {
         window.addEventListener('scroll', handleScroll);
+        window.addEventListener('resize', handleResize);
 
         return () => {
             window.removeEventListener('scroll', handleScroll);
+            window.removeEventListener('resize', handleResize);
         };
     }, []);
 
+    const closeMenu = () => {
+        setIsMenuOpen(false);
+    };
+
+    const handleResize = () => {
+        if (window.innerWidth >= 768) {
```

Comment:
It might be better to have a set of constants which define breakpoints under the `constants` folder or something

Reply:
Should I do this for just md (768px), or should I include other sizes that Tailwind uses like lg (1024px)?

Reply:
I'd say other sizes as well